ArduCopter/fence.cpp

```diff
@@ -4,29 +4,57 @@
 
 #if AP_FENCE_ENABLED
 
-// fence_check - ask fence library to check for breaches and initiate the response
-// called at 1hz
-void Copter::fence_check()
+// async fence checking io callback at 1Khz
+void Copter::fence_run_checks()
```

Comment:
I'm not a huge fan of the names, "fence_run_checks" vs "fence_checks".  these two names are different but they don't really make it clear what's different about them.  So for example, "fence_checks_async" would be better.  Other names might also be fine, this is just a suggestion

Reply:
Fixed thanks